container_workflow_tool/distgit.py

```diff
@@ -439,21 +439,21 @@ def _clone_downstream(self, component, branch):
             self.logger.info("Using existing downstream repo: " + component)
             repo = Repo(component)
         else:
-            ccomponent = "container/" + component
+            hostname_url = u._get_hostname_url(self.conf)
```

Comment:
Branch here depending on whether hostname_url is present and fallback to `packager` if it is not.